nifi-nar-bundles/nifi-standard-bundle/nifi-standard-processors/src/main/java/org/apache/nifi/processors/standard/GetFile.java

```diff
@@ -270,6 +280,9 @@ private FileFilter createFileFilter(final ProcessContext context) {
         return new FileFilter() {
             @Override
             public boolean accept(final File file) {
+                if (file.isDirectory()) {
+                    return false;
```

Comment:
does filtering work differently with the streams?  I ask because we want to be able to support the same depth/recursive searches and this filter seems like it would stop at dir level.

Reply:
If you look at the performListing, the filter is applied after the walk, with no feedback mechanism, so the additional filter just prevents Directories being returned as entities for GetFile to process.